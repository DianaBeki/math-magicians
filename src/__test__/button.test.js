import React from 'react';
import { render } from '@testing-library/react';
import Button from '../components/button';

// This will generate/update snapshots for the Button component
test('Button component works correctly', () => {
  const { asFragment } = render(<Button label="Click Me" onClick={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});
