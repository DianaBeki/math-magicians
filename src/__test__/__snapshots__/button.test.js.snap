// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component works correctly 1`] = `
<DocumentFragment>
  <button
    class="button"
    type="button"
  >
    Click Me
  </button>
</DocumentFragment>
`;
